yourgov/components/Staff/AccessRequestsTable/AccessRequestsTable.tsx

```diff
@@ -0,0 +1,343 @@
+import { useMemo, useState } from 'react';
+import {
+	Table,
+	TableBatchActionsBar,
+	TableBatchActionsTitle,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableHeader,
+	TableRow,
+	TableWrapper,
+} from '@ag.ds-next/react/table';
+import { TextLink } from '@ag.ds-next/react/text-link';
+import { StatusBadge } from '@ag.ds-next/react/status-badge';
+import { Button, ButtonGroup } from '@ag.ds-next/react/button';
+import { Flex } from '@ag.ds-next/react/flex';
+import { Checkbox } from '@ag.ds-next/react/checkbox';
+import { VisuallyHidden } from '@ag.ds-next/react/src/a11y';
+import { Stack } from '@ag.ds-next/react/stack';
+import { Box } from '@ag.ds-next/react/box';
+import { mapSpacing } from '@ag.ds-next/react/core';
+import { Divider } from '@ag.ds-next/react/divider';
+import { AvatarIcon } from '@ag.ds-next/react/icon';
+import { Heading } from '@ag.ds-next/react/heading';
+import { Text } from '@ag.ds-next/react/text';
+import { SectionAlert } from '@ag.ds-next/react/section-alert';
+import { plural } from '../lib/utils';
+import { useStaffGlobalState } from '../StaffProvider';
+import { StaffMemberAccessRequest } from '../lib/accessRequests';
+import { ModalConfirmApproveAccess } from './ModalConfirmApproveAccess';
+import { ModalConfirmRejectAccess } from './ModalConfirmRejectAccess';
+import { ModalUnavailableFeature } from './ModalUnavailableFeature';
+
+export const AccessRequestsTable = () => {
+	const { accessRequestsDelete, accessRequestsGetState, accessRequestsUpdate } =
+		useStaffGlobalState();
+	const [rowSelections, setRowSelections] = useState<
+		StaffMemberAccessRequest['id'][]
+	>([]);
+	const [itemWithActiveAction, setItemWithActiveAction] =
+		useState<StaffMemberAccessRequest>();
+
+	const [approveModalOpen, setApproveModalOpen] = useState(false);
```

Comment:
Not important, but perhaps we should we be using `useTernaryState` so we have specific open and close functions